queries.sql

```diff
@@ -28,4 +28,64 @@ WHERE name != 'Gabumon';
 
 SELECT *
 FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-----
+BEGIN;
+    UPDATE animals
+SET species = 'unspecified';
+    ROLLBACK;
+END;
+
+-----
+BEGIN;
+    UPDATE animals SET species = 'digimon' WHERE name like '%mon';
+    UPDATE animals SET species = 'pokemon' WHERE species is null;
+    COMMIT;
+END;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=species%20already%20set.-,Verify%20that%20changes%20were%20made.,Verify%20that%20changes%20persist%20after%20commit.,-Take%20a%20screenshot), Please verify that changes were made and verify that changes persist after commit.